.github/workflows/release.yml

```diff
@@ -103,6 +104,38 @@ jobs:
           fi
           git tag -l -n --format="%(contents:subject)" $(git describe --tags --abbrev=0) > TAG-COMMENT.txt
 
+      - name: MSI
+        shell: bash
+        if: matrix.os == 'windows-latest'
+        run: |        
+          # Return a qldb.exe
+          cargo fetch && cargo build --release  
+          
+          strip "target/${{ matrix.target }}/release/${{ matrix.bin }}"
+          
+          cargo install cargo-wix
+          
+          # create folder in `target/wix`
+          # do not do this as we now have a wix/main.wxs   #cargo wix init 
+               
+          # Then create MSI
+          cargo wix 
+          
+          cd target/wix
+          echo "Before Updating"
+          ls
+          
+          # rename msi file to "qldb.msi"
+          for file in *.msi; do
```

Comment:
Is the name of the generated msi not known? Also, can we use qldb_shell.msi as the resulting name?

Reply:
We should be using the naming convention as described on L85 (basically same name as tar.gz files)

Reply:
In that case we shouldn't need a for loop - we should just know the file and rename it?

Reply:
@Dominitio What is the msi file name it generates when you call `cargo wix`? Does it consistently create the same file name everytime?

Reply:
It generates this filename:  amazon_qldb_shell-0.1.0-x86_64.msi